xcode/Subconscious/Shared/Components/Notebook/NotebookNavigationView.swift

```diff
@@ -12,18 +12,23 @@ struct NotebookNavigationView: View {
     @ObservedObject var store: Store<NotebookModel>
     
     @Environment (\.colorScheme) var colorScheme
+    @Namespace var namespace
     
     func notify(_ notification: EntryNotification) -> Void {
         switch notification {
         case let .requestDetail(entry):
-            store.send(
-                .pushDetail(
-                    MemoEditorDetailDescription(
-                        address: entry.address,
-                        fallback: entry.excerpt.description
+            if app.state.isModalEditorEnabled {
+                app.send(.editorSheet(.editEntry(entry)))
+            } else {
```

Comment:
If the setting is enabled notes edited from the notebook will launch in the sheet.